core/src/main/scala/anorm/ToSql.scala

```diff
@@ -122,4 +121,7 @@ object ToSql {
 
     sql._1.toString -> sql._2
   }
+
+  /** Fallback when no instance is available. */
+  def missing[A] = null
```

Comment:
Do you want to put a type annotation on this so it's not inferred as `AnyRef`? E.g. `def missing[A]: ToSql[A] = null`.

Reply:
That's a workaround as with explicit type this triggers a MiMa error:

```
... where it is anorm.ToSql rather than scala.runtime.Null#
```

Just added a `@deprecated` to advice not to use it outside from Anorm itself.